### PR TITLE
udpate README.jp.md

### PR DESCRIPTION
**Pull request by tanabe**
Hello.

I also enhanced Japanese translations of README.jp.md.

thanks.

### CODE REVIEW

File: README.jp.md [hunk 1]
```diff
@@ -51,153 +53,154 @@ aptを使って、こちらのパッケージをインストールします。
...
 
-遊ぶ時にプレイヤー１の叩くキーはデフォールト[D F J K]に設定されています。Dは左カツで、Fは左ドンと言う感じになります。プレイヤー２の場合は[End PageDown Numpad8 Numpad9]に設定されています。
+遊ぶ時にプレイヤー 1 の叩くキーはデフォールト [D F J K] に設定されています。D は左カツで、F は左ドンと言う感じになります。プレイヤー 2 の場合は [End PageDown Numpad8 Numpad9] に設定されています。
 
 #### キーを変更する
-キーを変更するには「プレイヤー」メニューに移動して、「キーマップを設定する」オプションを選択します。キーを追加したい場合はESCAPE以外いずれかのキーをおすとそのキーは追加されます。
+キーを変更するには「プレイヤー」メニューに移動して、「キーマップを設定する」オプションを選択します。キーを追加したい場合は ESC 以外いずれかのキーを押すと、そのキーが設定されます。
 
-キーの設定は_keybinds.json_に登録しています。このファイルはテキストエディタで変更しても平気です。設定はプレイヤーナンバーに関してのみです。特定のプレイヤーに設定していません。
+キーの設定は _keybinds.json_ に登録しています。このファイルはテキストエディタで変更しても平気です。設定は選択したプレイヤーのみ有効です。他のプレイヤーには影響しません。
```
Comment (gtensha) [inline README.jp.md]: This is good, but the last two sentences may not convey what I wanted to say. Sorry for the vagueness.
「設定は選択したナンバーのプレイヤーのみ有効です」でどうでしょうか。
For example, when you set keybinds for player 2, the keybindings will always be the same no matter which player name enters as player 2 in the game session.

File: README.jp.md [hunk 2]
```diff
@@ -51,153 +53,154 @@ aptを使って、こちらのパッケージをインストールします。
...
 
 ### キーボードの使い方
-ドラムを叩くにはキーボードを使います。マウスはサポートされていません。
+ドラムを叩くにはキーボードを使います。マウスはサポートしていません。
 
-メニューを動かすには矢印キーを使います。ENTERをおすと決定し、ESCAPEなら前のメニューに戻ります。タイトル画面まで戻りさらにもう一度ESCAPEをおすとゲームを終了します。メニューカテゴリから選択するにはTABをおします。
+メニューを動かすには矢印キーを使います。ENTER キーを押すと決定し、ESC キーで前のメニューに戻ります。タイトル画面まで戻りさらにもう一度 ESC キーをおすとゲームを終了します。メニューを選択するには TAB を押します。
 
-遊ぶ時にプレイヤー１の叩くキーはデフォールト[D F J K]に設定されています。Dは左カツで、Fは左ドンと言う感じになります。プレイヤー２の場合は[End PageDown Numpad8 Numpad9]に設定されています。
+遊ぶ時にプレイヤー 1 の叩くキーはデフォールト [D F J K] に設定されています。D は左カツで、F は左ドンと言う感じになります。プレイヤー 2 の場合は [End PageDown Numpad8 Numpad9] に設定されています。
```
Comment (gtensha) [inline README.jp.md]: As we had this change before;
左カツ → 左カッ？

**Review by gtensha — CHANGES_REQUESTED** [review]
Hello, thank you for your continued contributions. I really appreciate it.
I'm thoroughly satisfied with this translation. It really raises the quality of the project.

This time I have two things I'd like you to look at before I merge your changes. Let me know if anything is unclear. Thanks!

File: README.jp.md (hunk 2 above)
Comment (tanabe) [reply README.jp.md]: Yes, that's right!

**Review by gtensha — APPROVED** [review]
Excellent. Thank you very much!

**Pull request merged**